src/model/tests/ModelUnitTest.cpp

```diff
@@ -364,6 +363,29 @@ void checkReducedModel(const Model & model)
         fullRNEA.runRNEA(fullPos,fullVel,fullAcc,fullTrqs);
         reducedRNEA.runRNEA(reducedPos,reducedVel,reducedAcc,reducedTrqs);
 
+        std::cerr << "Full Model: << " << model.toString() << std::endl;
+        std::cerr << "Reduced Model: << " << reducedModel.toString() << std::endl;
```

Comment:
I tought I had already commented this, this is a debug output taht should be removed.